ArduCopter/mode_zigzag.cpp

```diff
@@ -298,4 +298,9 @@ bool ModeZigZag::calculate_next_dest(uint8_t dest_num, bool use_wpnav_alt, Vecto
     return true;
 }
 
+bool ModeZigZag::requires_terrain_failsafe() const
```

Comment:
zigzag mode actually handles the terrain failures itself so we can remove this bit

Reply:
Removed.